tests/test_response_processor.py

```diff
@@ -134,7 +135,8 @@ def test_validate_returns_400(self):
             self._process()
 
             # receipt is not called
-            self.assertFalse(self.rp.send_receipt.called)
+            self.assertTrue(self.rp.send_receipt.called)
+            self.assertFalse(self.rp.make_receipt.called)
```

Comment:
Should we add a test for an 'lms' survey like the census and feedback specific ones?